recipes/catch2/2.x.x/conandata.yml

```diff
@@ -44,7 +44,7 @@ sources:
   2.13.6:
     url: https://github.com/catchorg/Catch2/archive/v2.13.6.tar.gz
     sha256: 48dfbb77b9193653e4e72df9633d2e0383b9b625a47060759668480fdf24fbd4
-  "2.13.7":
+  2.13.7:
```

Comment:
Not related to the review, but wanted to make this file consistent

Reply:
that's fine